currency_converter.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import sys
+
+from forex_python.converter import CurrencyCodes
+from forex_python.converter import CurrencyRates
+from forex_python.converter import RatesNotAvailableError
+
+
+code_dictionary = \
```

Comment:
why `\` everywhere?

why not:

```
code_dictionary = {
    u'Fr.': 'CHF',
    u'Kr':  'DKK'
}
```

from which language do you have this notation? :)